ledger/participant-integration-api/src/main/resources/db/migration/h2database-appendonly/V1__Append_only_schema.sql

```diff
@@ -24,7 +24,7 @@ CREATE TABLE configuration_entries (
     recorded_at BIGINT NOT NULL,
     submission_id VARCHAR NOT NULL,
     typ VARCHAR NOT NULL,
-    configuration BYTEA NOT NULL,
+    configuration BINARY LARGE OBJECT NOT NULL,
```

Comment:
apparently H2 changed behavior with original BYTEA (boiling down to BINARY VARYING): limited to 1Mb. To erase this limit we switch to BINARY LARGE OBJECT everywhere needed